geode-redis/src/main/java/org/apache/geode/redis/internal/gfsh/RedisCommandFunction.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.redis.internal.gfsh;
+
+import org.apache.geode.cache.execute.Function;
+import org.apache.geode.cache.execute.FunctionContext;
+import org.apache.geode.cache.execute.FunctionService;
+import org.apache.geode.internal.cache.InternalCache;
+import org.apache.geode.management.internal.functions.CliFunctionResult;
+import org.apache.geode.redis.internal.GeodeRedisService;
+
+public class RedisCommandFunction implements Function<Boolean> {
```

Comment:
For consistency with other cli-related functions we should `extend CliFunction` instead of `implements Function`. That class exposes an `execute` that just needs to return a `CliFunctionResult`. `CliFunction` also extends `InternalFunction` which is more secure (it cannot be user-invoked in any way - for example via gfsh).

Reply:
I updated it, let me know what you think.